examples/00-load/read_dolfin.py

```diff
@@ -32,7 +32,7 @@
 
 # %%
 # Now we can work on and plot that Dolfin mesh.
-qual = dolfin.compute_cell_quality()
+qual = dolfin.cell_quality()
```

Comment:
```suggestion
qual = dolfin.cell_quality('scaled_jacobian')
```

The generated images from this example have changes because the new default measure is `'shape'`. We can add `'scaled_jacobia'` explicitly to keep the previous images.